test/Tests.Infrastructure/TestBase.cs

```diff
@@ -182,6 +186,122 @@ public void DoNotReuseServer(IDictionary<string, string> customSettings = null)
             _doNotReuseServer = true;
         }
 
+        protected static TestCertificatesHolder _selfSignedCertificates;
+
+        protected TestCertificatesHolder GenerateAndSaveSelfSignedCertificate(bool createNew = false, [CallerMemberName] string caller = null)
```

Comment:
This is a copy of a method from RavenTestBase.Certificates. You should use it or move it to this class, depending why you need it here.

Reply:
fixed